crates/reward/src/lib.rs

```diff
@@ -43,35 +43,18 @@ pub mod pallet {
         type CurrencyId: Parameter + Member + Copy + MaybeSerializeDeserialize + Ord;
     }
 
-    #[derive(Encode, Decode, Clone, Copy, PartialEq, Debug)]
-    pub enum RewardPool<AccountId> {
```

Comment:
This is no longer necessary because the fee pallet will re-distribute rewards to the Vault's staking pool instead of `Local(AccountId)`.